### PR TITLE
SRCHX-1142: Filter selection bug fix

### PR DESCRIPTION
**Pull request by rehanabbasi**
Resolves SRCHX-1142

## Description

Changes in this PR are meant to fix a reported prod bug related to filter selection. Apparently the filter selection caused an uncaught error on the UI side where we were trying to split a parsed array by `,`. This prevented the retention of updated search term in the filters component and causing unexpected search behavior on the UI side.

## Checks

**Have you written any necessary unit tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Have you added or updated any necessary integration tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Browsers tested on:**

- [ ] Chrome
  - [ ] Mobile
- [ ] Safari
  - [ ] Mobile
- [x] Firefox
  - [ ] Mobile
- [ ] Edge
- [ ] IE11
- [ ] Not applicable

### CODE REVIEW

**Review by gbassham — APPROVED** [review]

File: src/app/asset-filters/asset-filters.component.ts [hunk 1]
```diff
@@ -152,7 +152,7 @@ export class AssetFilters {
               }
               // Check if there are multiple selected Geo facets.
               if(paramName === 'geography') {
-                parsedParam = parsedParam.split(',')
+                parsedParam = parsedParam
               }
```
Comment (daneah) [inline src/app/asset-filters/asset-filters.component.ts:156]: It seems like this whole block could just go away instead? (`parsedParam = parsedParam` seems not to have any effect)

**Review by daneah — COMMENTED** [review]
It seems like the whole

**Review by NazimAli2017 — APPROVED** [review]
Aside from Danes suggestion, looks good 👍

File: src/app/asset-filters/asset-filters.component.ts (hunk 1 above)
Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.ts:156]: Now that I have read your comment, I feel silly for not thinking about this earlier.

Comment (rehanabbasi) [reply src/app/asset-filters/asset-filters.component.ts]: @daneah change made in the commit `4e16fa5`.

**Review by daneah — APPROVED** [review]

**Pull request merged**